grouper/capabilities.py

```diff
@@ -1,11 +0,0 @@
-import bittle
```

Comment:
looks like this is the only usage of `bittle`, mind nuking it from All The Places?